src/Engines/AlgoliaEngine.php

```diff
@@ -134,4 +148,48 @@ public function map($results, $model)
             return $models[$hit[$model->getKeyName()]];
         });
     }
+
+    /**
+     * Sets settings to Algolia index.
+     * https://www.algolia.com/doc/api-client/php/settings#set-settings
+     *
+     * @param  \Illuminate\Database\Eloquent\Model  $model
+     * @param  mixed  $settings
+     * @return void
+     */
+    public function setSettings($model, $settings)
```

Comment:
Model should be type-hinted maybe?
`public function setSettings(Model $model, $settings)`